docs/platforms/javascript/common/migration/v7-to-v8/v8-opentelemetry.mdx

```diff
@@ -0,0 +1,93 @@
+---
+title: OpenTelemetry Support
```

Comment:
Eventually we should move this under `javascript/guides/node/performance/instrumentation/opentelemetry/` and similar, but I think this is a good place for this to live for now while v8 is in a beta state.